lib/dotcom_web/components/system_status/status_row_heading.ex

```diff
@@ -58,6 +63,45 @@ defmodule DotcomWeb.Components.SystemStatus.StatusRowHeading do
     """
   end
 
+  defp subheading(%{alerts: []} = assigns), do: ~H""
+
+  defp subheading(%{status: :station_closure} = assigns) do
```

Comment:
Much of this module will likely need to be changed, including undoing most (all?) of the changes from https://github.com/mbta/dotcom/pull/2536. The hi-fi designs changed from a design that was well-suited to a grid to a design that was better-suited to a flexbox.